footprints/templates/main/gallery_view.html

```diff
@@ -1,23 +1,48 @@
 <!--Gallery View for search.html-->
 
+<div class="gallery-display" hidden>
```

Comment:
We use a standard of four space characters as indentation instead of two at the CTL - for consistency, so I would adjust your editor to account for this with HTML files.